documentation/list_games.md

```diff
@@ -0,0 +1,126 @@
+# List games
+
+Returns json data about the list of games that a user has created.
+
+- **URL:**
+
+  /api/game/list
+
+- **Method:**
+
+  `GET`
+
+- **URL Params:**
+
+  None
+
+- **Data Params:**
+
+  **Required:** Size
+
+  `size: [unsigned int]` - Must be between 1 and 100
+
+  50
+
+  **Optional:** Offset - how many games to skip
+
+  `offset: [unsigned int]`
+
+  0
```

Comment:
This either should be changed into a POST request or the body params should be moved to the URI